src/components/footer.tsx

```diff
@@ -3,7 +3,7 @@ import Link from "next/link";
 
 export default function Footer() {
   return (
-    <div className="Footer flex h-24 w-full items-end justify-between border-t border-midnight/20 bg-apricot/5 px-16 py-8">
```

Comment:
Sad to see you remove the divider here. I think it really helps to differentiate the footer from the page